features/apimgt/org.wso2.carbon.apimgt.rest.api.store.feature/src/main/resources/api-docs/docs/models/API.html

```diff
@@ -184,6 +184,21 @@ <h2>API</h2>
                         <td class="parameter"> <p class="marked">null</p></td>
                         <td class="parameter"> <p class="marked">[&quot;http&quot;,&quot;https&quot;]</p></td>
                     </tr>
+                    <tr>
+                        <td class="parameter">
+                            <p>authorizationHeader</p>
+                            <p class="param-required">
+                                
```

Comment:
Please remove the additional new lines.

Reply:
These are generated code. Do we need to modify these.

Reply:
Then it's fine.